docs/api-reference/collection.md

```diff
@@ -102,18 +115,21 @@ This method returns a list of all models in the collection. The order is not gua
 **Note:** This method will probably be deprecated in the future in favor of the `findAll` method.
 
 ### filter
+
 ```typescript
 filter(test: TFilterFn): Array<PureModel>
 ```
 
 A method used to filter through the models, similar to the [Array filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter). Keep in mind that this will go trough all model types and you should check the model type also.
 
 Alternatively, you can use a combination of `findAll` and the native `filter` method instead:
+
 ```typescript
-data.findAll(User).filter((user) => user.age >= 18);
+data.findAll(User).filter(user => user.age >= 18);
```

Comment:
is this because of the default prettier config or something?

Reply:
It might be, I will check that out.